muttlib.c

```diff
@@ -1313,12 +1314,14 @@ int mutt_save_confirm(const char *s, struct stat *st)
   {
     if (Confirmappend)
     {
-      snprintf(tmp, sizeof(tmp), _("Append messages to %s?"), s);
-      rc = mutt_yesorno(tmp, MUTT_YES);
-      if (rc == MUTT_NO)
-        ret = 1;
-      else if (rc == MUTT_ABORT)
-        ret = -1;
+	/* Maybe waste some spaces here */
+        snprintf(tmp, sizeof(tmp), _("Append messages to %s?"), s);
+        rc = mutt_yesorno(tmp, MUTT_YES);
+        if (rc == MUTT_NO)
+          ret = 1;
+        else if (rc == MUTT_ABORT)
+          ret = -1;
+
     }
```

Comment:
Please can you replace all the `<tab>`s with spaces.
You can run `clang-format -i FILE` to fix most formatting issues.